### PR TITLE
chore:  move around team and contributors

### PR DESCRIPTION
**Pull request by BambooXLotus**

### CODE REVIEW

**Comment by BambooXLotus** [comment]
move around team and contributors

**Comment by DistractionBoy** [comment]
could you remove Weston Nelson from the top, and replace his spot by Dicaso's? Sorry, now that i'm looking at it i realize i was kinda all over the place in my suggestions for who to put where. but basically, 

Core Team -> me, Dicaso, JasonFactor

Contributors -> Daru, and you

File: projects/marketing-site/components/Team/CoreTeam.tsx [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
     linkedinUrl: null,
   },
   {
-    name: "Weston Nelson",
-    role: "Treasury Management",
-    imageUrl: weston,
-    twitterUrl: "https://twitter.com/westonnelson",
+    name: 'Jason Factor',
+    role: 'anhfactor',
+    imageUrl: jason,
```
Comment (DistractionBoy) [inline projects/marketing-site/components/Team/CoreTeam.tsx:26]: could we still use https://avatars.githubusercontent.com/u/13186215?v=4 

or is that too small for the space? it'd be cool if we could not have to keep that image file in the repo if we don't need to.

Comment (BambooXLotus) [reply projects/marketing-site/components/Team/CoreTeam.tsx:26]: I get this error when I run it locally with that.
image

File: projects/marketing-site/components/Team/CoreTeam.tsx [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
     imageUrl: dicaso,
-    twitterUrl: "https://twitter.com/Dicaso5",
+    twitterUrl: 'https://twitter.com/Dicaso5',
     linkedinUrl: null,
   },
   {
-    name: "Weston Nelson",
-    role: "Treasury Management",
-    imageUrl: weston,
-    twitterUrl: "https://twitter.com/westonnelson",
```
Comment (anhfactor) [inline projects/marketing-site/components/Team/CoreTeam.tsx:28]: Can we use github for my profile ?. I'd like to show github than twitter. 
This is my github: https://github.com/anhfactor

File: projects/marketing-site/components/Team/CoreTeam.tsx (hunk 1 above)
Comment (anhfactor) [reply projects/marketing-site/components/Team/CoreTeam.tsx:26]: @BambooXLotus  To fix this error, in "./projects/marketing-site/components/Team/CoreTeam.tsx", add additional "width={80}" and "height={80}" properties in <Image> tag like this

`<Image
                    className="mx-auto h-40 w-40 rounded-full xl:h-56 xl:w-56"
                    src={person.imageUrl}
                    alt=""
                    priority
                    width={80}
                    height={80}
                  />`
I have checked and it worked!

File: projects/marketing-site/components/Team/CoreTeam.tsx [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
+    name: 'Jason Factor',
```
Comment (anhfactor) [inline projects/marketing-site/components/Team/CoreTeam.tsx:24]: Can we use github for my profile ?. I'd like to show github than twitter.
This is my github: https://github.com/anhfactor

File: projects/marketing-site/components/Team/CoreTeam.tsx (hunk 1 above)
Comment (BambooXLotus) [reply projects/marketing-site/components/Team/CoreTeam.tsx:26]: It works but it seems to be getting a low quality Image. I will look more into this tommorow.

image

Comment (DistractionBoy) [reply projects/marketing-site/components/Team/CoreTeam.tsx:26]: let's just leave it as-is. i don't like pixelated images / low quality images

File: projects/marketing-site/components/Team/CoreTeam.tsx (hunk 3 above)
Comment (DistractionBoy) [reply projects/marketing-site/components/Team/CoreTeam.tsx:24]: Yeah! that's cool!

**Review by DistractionBoy — APPROVED** [review]
LGTM

**Pull request merged**